README.md

```diff
@@ -33,3 +39,4 @@ Alternatively, you are free to vendor directly a copy of Darkdetect in your app.
 - This software is licensed under the terms of the 3-clause BSD License.
 - This package can be installed on any operative system, but it will always return `None` unless executed on a version of macOS or Windows that supports Dark Mode. This package is designed to work also with older versions of macOS and Windows and in those cases it will also return `None`. Detection of the dark menu bar and dock option (available from macOS 10.10) is not supported.
 - [Details](https://stackoverflow.com/questions/25207077/how-to-detect-if-os-x-is-in-dark-mode) on the detection method used on macOS.
+- [Details](https://askubuntu.com/questions/1261366/detecting-dark-mode#comment2132694_1261366) on the experimental detection method used on Linux, which just attempts to check if the GTK theme ends in `'-dark'` by making a call to `gsettings`.
```

Comment:
Maybe you could just stop after `Linux` here, as the link already provides details.